docs/crr-to-aws-s3.md

```diff
@@ -75,3 +75,334 @@ env_replication_endpoints:
 
 * Backbeat updates the source object's metadata to store AWS' version id and set
   replication status as `COMPLETED`
+
+## Setting up Cross-Region Replication (CRR) to AWS
+
+This section contains steps to set up cross-region replication from Scality S3
+to Amazon Web Services S3 using the Backbeat replication engine. Two subsections
+are provided with different methods: the subsection "Basic Example" uses a
```

Comment:
The explanation about the subsections may go directly into their respective "Basic" and "Detailed" subsections introductory paragraph IMO. And instead of "Basic" and "Detailed", names like "Half-Automated" and "Manual" (for example) seem more appropriate to me.

Reply:
I like that idea! Updated.